x/bank/keeper/keeper.go

```diff
@@ -0,0 +1,32 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"github.com/axelarnetwork/axelar-core/x/bank/types"
+)
+
+// BankKeeper wraps the bankkeeper.BaseKeeper.
+type BankKeeper struct {
+	types.BankKeeper
+}
+
+// NewBankKeeper returns a new BankKeeper.
+func NewBankKeeper(bk types.BankKeeper) BankKeeper {
+	return BankKeeper{bk}
+}
+
+// SendCoins transfers amt coins from a sending account to a receiving account.
+// An error is returned upon failure, or when the from/to address is blocked.
+func (k BankKeeper) SendCoins(ctx sdk.Context, fromAddr sdk.AccAddress, toAddr sdk.AccAddress, amt sdk.Coins) error {
```

Comment:
Looks good to me. We also make use of `SendCoinsFromAccountToModule` and `SendCoinsFromModuleToAccount` but their current usage don't involve sender/recipient accounts that could be blocked.

Reply:
We can overwrite the behaviour of those two later if needed. The biggest point of this PR is to develop the framework for overwriting SDK stuff.